tests/src/Traits/FacetTestGeneratorTrait.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+declare(strict_types=1);
+
+namespace OpenEuropa\Tests\EuropaSearchClient\Traits;
+
+use OpenEuropa\EuropaSearchClient\Model\Facet;
+use OpenEuropa\EuropaSearchClient\Model\FacetValue;
+
+trait FacetTestGeneratorTrait
+{
+    /**
+     * @param int $count
+     * @return array
+     *   An array with two items:
+     *   0: A list of facet values as raw array.
+     *   1: The same list as model object.
+     */
+    protected function generateTestingFacetValues(int $count): array
+    {
+        $facetValuesAsArray = [];
+        for ($i = 0; $i < $count; $i++) {
+            $toShuffle = md5(serialize($facetValuesAsArray));
+            $facetValuesAsArray[] = [
+                'apiVersion' => str_shuffle($toShuffle),
+                'count' => rand(1, 100000),
+                'rawValue' => str_shuffle($toShuffle),
+                'value' => str_shuffle($toShuffle),
+            ];
+        }
+
+        $facetValuesAsObject = array_map(
+            [$this, 'generateTestingFacetValue'],
+            $facetValuesAsArray
+        );
+
+        return [$facetValuesAsArray, $facetValuesAsObject];
```

Comment:
Just return one of them, and expose a method that does the conversion?

Reply:
done